src/utils.jl

```diff
@@ -231,3 +231,77 @@ function intersectparameters(a::Point{Dim,T}, b::Point{Dim,T}, c::Point{Dim,T},
 
   λ₁, λ₂, r, rₐ
 end
+
+"""
+bresenham(ind1::CartesianIndex{2}, ind2::CartesianIndex{2})
+
+Compute the Bresenham's line algorithm.
+
+## References
+
+* Bresenham's line algorithm - Wikipedia (https://en.wikipedia.org/wiki/Bresenham's_line_algorithm)
+"""
+function bresenham(ind1::CartesianIndex{2}, ind2::CartesianIndex{2})
```

Comment:
this function should have a different signature, (grid, start, finish) because sometimes start and finish may live outside the grid and we need to check that